### ISSUE TITLE
Compliance Profile inheritence does not work with audit cookbook

### ISSUE THREAD

**Issue by yvovandoorn** [chef-cookbooks/audit#38]
### Cookbook version

0.6.0 
### Chef-client version

12.9.41
### Platform Details

RHEL 7.2
### Scenario:

Attempting to run an audit profile that inherits another profile on the compliance server
### Steps to Reproduce:

Run the audit cookbook with a compliance policy that is inherited
### Expected Result:

Audit cookbook passes
### Actual Result:

`* compliance_profile[cis-rhel7-level1-lite] action execute
    * chef_gem[inspec] action install (up to date)
[2016-05-04T11:33:00-04:00] WARN: Using inspec version: (0.20.1)
    - install/update inspec
    ================================================================================
    Error executing action`execute` on resource 'compliance_profile[cis-rhel7-level1-lite]'
    ================================================================================

```
RuntimeError
------------
You must supply a --profiles-path to inherit from other profiles.

Cookbook Trace:
---------------
/var/chef/cache/cookbooks/audit/libraries/profile.rb:111:in `block (2 levels) in <class:ComplianceProfile>'
/var/chef/cache/cookbooks/audit/libraries/profile.rb:94:in `block in <class:ComplianceProfile>'

Resource Declaration:
---------------------
# In /var/chef/cache/cookbooks/audit/recipes/default.rb

 30:   compliance_profile p do
 31:     owner o
 32:     server server
 33:     token token
 34:     inspec_version node['audit']['inspec_version']
 35:     action [:fetch, :execute]
 36:   end
 37: end

Compiled Resource:
------------------
# Declared in /var/chef/cache/cookbooks/audit/recipes/default.rb:30:in `block in from_file'

compliance_profile("cis-rhel7-level1-lite") do
  action [:fetch, :execute]
  updated true
  retries 0
  retry_delay 2
  default_guard_interpreter :default
  declared_type :compliance_profile
  cookbook_name "audit"
  recipe_name "default"
  owner "thomasrgcheforg"
  inspec_version "latest"
  profile "cis-rhel7-level1-lite"
end

Platform:
---------
x86_64-linux
```

`

**Comment by chris-rock** [comment]
related to https://github.com/chef/inspec/issues/533

**Comment by jerryaldrichiii** [comment]
+1 to this bug.

Profile inheritance works if using the Compliance WebUI (SSH) to perform the scan but not when using `chef-client` and this cookbook.

**Comment by chris-rock** [comment]
This should be fixed with 3.1. Please re-open if this continues to be an issue.
